docs/guide/migration/overview.md

```diff
@@ -0,0 +1,36 @@
+# Changes from Apostrophe 2
```

Comment:
This page is mostly unchanged from the original ["Coming from 2.x" ](https://v3.docs.apostrophecms.org/guide/upgrading.html) page. The breaking changes section is now in the upgrading.md file.